packages/cli/templates/vue3/src/components/ModalToggle.vue

```diff
@@ -8,9 +8,10 @@ defineProps<{
 </script>
 
 <template>
-  <div class="modal-toggle">
+  <div>
+    <h2>Modal</h2>
     <button
-      class="btn"
+      class="btn w-full"
```

Comment:
We could create a `Stack` component with

```
    display: flex;
    flex-direction: column;
    align-items: stretch;
```

And reuse that to stack elements on top of each other.

Same thing could be done with `Button`, `Divider`, and the headers.

Reply:
I don't think it's a good idea to start building a UI library inside these templates. We don't need such complexity where we should just demonstrate how to use it. You mentioned about having less component with App.vue and FieldPlugin.vue, but these are way more components.